{{cookiecutter.project_slug}}/frontend/README.md

```diff
@@ -1,37 +1,40 @@
-# Vite + React + Typescript + Eslint + Prettier + Jest(Testing Library)
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
-A starter for React with Typescript with the fast Vite and all static code testing with Eslint,formatting with Prettier and basic unit test template with Jest.
+## Getting Started
 
+First, run the development server:
 
-## Installation
+```bash
+npm run dev
+# or
+yarn dev
+# or
+pnpm dev
+# or
+bun dev
+```
```

Comment:
These instructions won't work with Tilt. We can simply say that Tilt will automatically deploy a NextJS container for development, and it is recommended not to run NextJS locally.